### PR TITLE
feat/10: system alerts provisioning code

### PR DESCRIPTION
**Pull request by rammohan-y**
implemented code for writing system_alerts

Note: 
Not calling writeSystemAlerts in setInterval like others, because the record to be inserted immediately
In writeSystemAlerts method, not checking for data.length > commitSize because the record to be immediately inserted

### CODE REVIEW

**Comment by vdharashive** [comment]
@davehorton can you pls review and merge

File: index.js [hunk 1]
```diff
@@ -89,6 +89,17 @@ const schemas = {
...
+  },
+  system_alerts: {
+    measurement: 'system_alerts',
+    fields: {
+      status: Influx.FieldType.STRING,
+      detail: Influx.FieldType.STRING,
+      host: Influx.FieldType.STRING
+    },
+    tags: [
+      'system_category'
```
Comment (davehorton) [inline index.js]: I would rather we named this "system_component".
Also, I would like to add an additional tag named "state", which will have a value of either 'offline' or 'online'.  That way we can also detect changes when a component goes offline or comes online.

File: index.js [hunk 2]
```diff
@@ -89,6 +89,17 @@ const schemas = {
       'account_sid',
       'application_sid'
     ]
+  },
+  system_alerts: {
+    measurement: 'system_alerts',
+    fields: {
+      status: Influx.FieldType.STRING,
```
Comment (davehorton) [inline index.js]: I think we can get rid of this in favor of having a tag 'state'.  Making it a tag value means we will be able to search on state values

**Review by davehorton — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (rammohan-y) [reply index.js]: @davehorton made changes as per your recommendations, thank you

**Review by davehorton — APPROVED** [review]

**Comment by davehorton** [comment]
published in v0.2.11

**Pull request merged**